tests/gold_tests/autest-site/trafficserver.test.ext

```diff
@@ -120,7 +120,7 @@ def MakeATSProcess(obj, name, command='traffic_server', select_ports=True):
     p.Env['PROXY_CONFIG_HOSTDB_STORAGE_PATH'] = runtime_dir
     # p.Variables.RUNTIMEDIR=runtime_dir
     p.Setup.MakeDir(runtime_dir)
-    p.Setup.Chown(runtime_dir, "nobody", "nobody",ignore=True) 
+    p.Setup.Chown(runtime_dir, "nobody", "nobody",ignore=True)
```

Comment:
Do you still need this? I thought the point was to make the chown more specific, as in the code section for the cache.db directory just below.